clients/web/apps/docs/src/styles/custom.css

```
@@ -1,420 +1,314 @@
 /* ─────────────────────────────────────────────
-   Corvus — Custom Starlight Theme
-   Modern dark design with gradient accents
+   Corvus — Nothing Design System for Starlight
+   Flat surfaces, border separation, typography hierarchy.
+   No shadows, no gradients, no blur, no glow.
    ───────────────────────────────────────────── */
 
+/* ── Font Loading — self-hosted via @fontsource ── */
+@import url("@fontsource-variable/space-grotesk");
```

Stylelint (17.6.0)

[error] Expected "url("@fontsource-variable/space-grotesk")" to be ""@fontsource-variable/space-grotesk"" (import-notation)

(import-notation)